packages/web/hooks/use-trade-token-in-config/index.ts

```diff
@@ -0,0 +1,323 @@
+import { useState } from "react";
+import { AmountConfig, IFeeConfig } from "@keplr-wallet/hooks";
+import { action, computed, makeObservable, observable, override } from "mobx";
+import { AppCurrency } from "@keplr-wallet/types";
+import { ChainGetter, IQueriesStore } from "@keplr-wallet/stores";
+import {
+  OptimizedRoutes,
+  Pool,
+  RoutePathWithAmount,
+} from "@osmosis-labs/pools";
+import {
+  CoinPretty,
+  Dec,
+  DecUtils,
+  Int,
+  IntPretty,
+  RatePretty,
+} from "@keplr-wallet/unit";
+
+export class TradeTokenInConfig extends AmountConfig {
```

Comment:
Same comments as in `use-slippage-config`

Reply:
Ditto